src/cli/snp.rs

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::backend::sev::Firmware;
+
+use anyhow::{anyhow, Context, Result};
+use std::fs::{self, remove_file};
+use std::io::{self, ErrorKind, Read, Seek};
+use std::os::unix::fs::OpenOptionsExt;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+/// SNP-specific functionality
+#[derive(StructOpt, Debug)]
+pub enum Command {
+    /// SNP VCEK related commands
+    Vcek(VcekCommand),
+}
+
+#[derive(StructOpt, Debug)]
+pub enum VcekCommand {
+    /// Print the VCEK certificate for this platform to stdout in PEM format
+    Show,
+    /// Print the VCEK certificate cache file used
+    ShowFile,
```

Comment:
Define "used" - it seems that we may use different paths on download and on query

Reply:
So, `update` might update another file in the hierarchy first, but _only_ if none of the others exist.

Reply:
clarified, that it prints the file used by the `show` command